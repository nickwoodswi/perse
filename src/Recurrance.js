import React, { Component } from 'react'

class Recurrance extends Component {
    static defaultProps = {
        recurrance: 1
      }
    render() {
        if (this.props.recurrance == 1) {
            return(<li>>>Every day</li>)
        }
        if (this.props.recurrance == 2) {
            return(<li>>>Every other day</li>)
        }
        if (this.props.recurrance == 3) {
            return(<li>>>Every 3 days</li>)
        }
        if (this.props.recurrance == 7) {
            return(<li>>>Every week</li>)
        }
    }
}

export default Recurrance