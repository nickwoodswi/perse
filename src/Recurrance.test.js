import React from 'react';
import { render } from '@testing-library/react';
import Recurrance from './Recurrance';
import { MemoryRouter } from 'react-router-dom'
import renderer from 'react-test-renderer'

test('renders Recurrance component', () => {
    render(<MemoryRouter><Recurrance /></MemoryRouter>);
});

it('Recurrance renders the UI as expected', () => {
    const tree = renderer
        .create(<MemoryRouter><Recurrance /></MemoryRouter>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});