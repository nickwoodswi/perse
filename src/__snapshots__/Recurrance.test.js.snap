// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Recurrance renders the UI as expected 1`] = `
<li>
  <b>
    EVERY DAY
  </b>
</li>
`;
